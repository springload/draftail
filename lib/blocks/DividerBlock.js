import React from 'react';

/**
 * An <hr/> in the editor.
 */
const DividerBlock = () => <hr className="Draftail-DividerBlock" />;

export default DividerBlock;
