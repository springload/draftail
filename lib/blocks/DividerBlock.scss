.#{$NS}DividerBlock {
    border: 0;
    background: $color-light-grey;
    height: 1px;
    margin: 0 0 20px;
}
