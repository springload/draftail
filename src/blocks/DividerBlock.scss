@use "../api/constants" as *;

.#{$DRAFTAIL}DividerBlock {
  border: 0;
  background: $color-light-grey;
  height: 1px;
  margin: 10px 0;

  @media (forced-colors: active) {
    background: CanvasText;
  }
}
