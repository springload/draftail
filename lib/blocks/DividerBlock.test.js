import React from "react";
import { shallow } from "enzyme";
import DividerBlock from "./DividerBlock";

describe("DividerBlock", () => {
  it("basic", () => {
    expect(shallow(<DividerBlock />)).toMatchSnapshot();
  });
});
