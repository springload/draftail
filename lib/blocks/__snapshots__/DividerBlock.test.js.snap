// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DividerBlock basic 1`] = `
<hr
  className="Draftail-DividerBlock"
/>
`;
